.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug about the Scala compiler or the standard library
+title: ''
+assignees: ''
+
+---
+
+## minimized code
+
+```Scala
+println("hello, world")
+```
+
+## Compilation output
+
+```scala
+// TODO add compilation output here
+```
+
+## expectation
```

Comment:
How about condensing the second & third sections into a single one, maybe "## Expected vs. actual results"

Let's err on the side of less structure rather than more structure, is my personal take this. I hate filling out forms like I'm at the DMV.

Happy to accede to consensus on this, though. Or to authority :-)

Reply:
Painting yellow lines on the road probably won't magically remove all traffic accidents. But in the aggregate, having them should improve the median driver's behavior when they are driving on unfamiliar roads. imho, separating out the expectation and asking people what they expect is similar: It makes the tone of the bug reports more constructive, setting up for a more collaborative fact finding.

For example, on sbt/sbt recent bug reports like https://github.com/sbt/sbt/issues/5444 and https://github.com/sbt/sbt/issues/5449 are good example of good bug reports.

I should also note that headers etc are meant as guidelines, and advanced users are free to ignore them as they see fit.

Reply:
I'm with Eugene on this one.  Even on the receiving end the little moment's thought having to figure out if what I'm writing are my expectations or if they're what I think should be fix/changed/etc, is very valuable for the resulting issue.

To me the biggest win is avoiding those tickets where the sense is "well, isn't it obvious what should've happened?!  Excercise for the reader."  It's good to be black-on-white about this.  Sometimes I even forget what I expected at the time, when I stumble upon old sbt issues.  It's net positive, IMHO.

Reply:
`## What the hell did you expect?`